src/main/kotlin/org/rust/lang/core/macros/Include.kt

```diff
@@ -0,0 +1,39 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.lang.core.macros
+
+import org.rust.lang.core.psi.RsLitExpr
+import org.rust.lang.core.psi.RsMacroCall
+import org.rust.lang.core.psi.RsMacroExpr
+import org.rust.lang.core.psi.RsPsiFactory
+import org.rust.lang.core.psi.ext.stringLiteralValue
+import org.rust.lang.core.resolve.DEFAULT_RECURSION_LIMIT
+import java.nio.ByteBuffer
+
+fun expandInclude(call: RsMacroCall): List<RsExpandedElement>? {
+    val psiFactory = RsPsiFactory(call.project)
+    val args = call.includeMacroArgument?.expr ?: return null
```

Comment:
The tests are really failed on this line. It's because `RsIncludeMacroArgument` isn't stubbed. You can read about stubs in out [docs](https://github.com/intellij-rust/intellij-rust/blob/master/ARCHITECTURE.md#stubs). It's not too hard to force some element to be stubbed. Just follow some commit that do the same thing for another element, e.g. [this](https://github.com/intellij-rust/intellij-rust/commit/8513a547cdce4b898613d667094f2c4df6b8d3b1). The element that have to be stubbed - `IncludeMacroArgument` (in `RustParser.bnf`).